.github/native-tests.json

```diff
@@ -63,7 +63,7 @@
         {
             "category": "Messaging2",
             "timeout": 80,
-            "test-modules": "reactive-messaging-amqp, reactive-messaging-rabbitmq, reactive-messaging-rabbitmq-dyn, reactive-messaging-pulsar",
+            "test-modules": "reactive-messaging-amqp, reactive-messaging-rabbitmq, reactive-messaging-rabbitmq-dyn, reactive-messaging-rabbitmq-devservices, reactive-messaging-pulsar",
```

Comment:
You don't need to add the test here as we don't really care testing native for it. The Dev Services are run in the exact same way.
Having the test run in JVM mode is enough.

(We are a bit careful about adding tests there as native compilation takes quite a lot of time)

Reply:
Okay, removed the native test.